halo1/model.py

```diff
@@ -334,9 +344,40 @@ def import_halo1_region_from_jms(jms, *, name="unnamed", scale=1.0, region_filte
 	scene = bpy.context.collection
 	scene.objects.link(region_obj)
 
+	# If the function was supplied with an armature object attempt to skin to it.
+
+	region_obj.parent = armature_obj
+
+	if skin_vertices and region_obj.parent.type == 'ARMATURE':
+		mod = region_obj.modifiers.new('armature', 'ARMATURE')
+		mod.object = armature_obj
+
+		# Create a vertex group for each bone.
+
+		for node in jms.nodes:
+			region_obj.vertex_groups.new(name=NODE_NAME_PREFIX+node.name)
+
+		vertex_groups = region_obj.vertex_groups
```

Comment:
Are we actually saving anything by breaking this value out into a variable? It seems like unnecessary added complexity, especially since you do call it directly off of `region_obj` right above this line.